egs/librispeech/ASR/distillation_with_hubert.sh

```diff
@@ -155,8 +155,16 @@ if [ $stage -le 2 ] && [ $stop_stage -ge 2 ]; then
     fi
     log "Downloading extracted codebook indexes to $codebook_download_dir"
     # Make sure you have git-lfs installed (https://git-lfs.github.com)
+    # The codebook indexes are generated using lhotse 1.11.0, to avoid
+    # potential issues, we recommend you to use the same version of lhotse
+    # to run `prepare.sh`.
+    lhotse_version=$(python3 -c "import lhotse; print(lhotse.version.__version__)")
+    if [ "$lhotse_version" != "1.11.0" ]; then
```

Comment:
Does it support `lhotse > 1.11.0`?

Reply:
I've only tested with `1.11.0`, newer versions could work.